### ISSUE TITLE
Trouble installing "directly from the PPA itself"

### ISSUE THREAD

**Issue by bennlich** [foss-project/green-recorder#111]
I'm kind of new to PPA and struggling to install green-recorder on my debian distro. Part of the problem seems to be that there is no disco distribution here? http://ppa.launchpad.net/fossproject/ppa/ubuntu/dists/

**Comment by mhsabbagh** [comment]
Which is fine. Just head to the following URLs and download the DEB packages and install them:

http://ppa.launchpad.net/fossproject/ppa/ubuntu/pool/main/g/green-recorder/
http://ppa.launchpad.net/fossproject/ppa/ubuntu/pool/main/p/pydbus/

Your "fix" will lead them (users) to the Launchpad page, but they can simply download packages from URLs above (maybe we should change the URLs to point to packages directly). Thanks for trying to help!

**Comment by bennlich** [comment]
Okay. Maybe pointing to the packages directly makes sense, though at that point I guess you're sort of rebuilding the launchpad list? Is the fear that the launchpad page may not last as long as the PPA server?

Anything that adds more explanation for those of us not familiar with PPA repo directory structures would be awesome IMO.

I found the launch pad page much easier to interpret--so much easier that I figured it was the link you had intended to use, and the other was a typo :-P

Thanks for replying with the above.
